python/grass/temporal/register.py

```diff
@@ -81,6 +119,14 @@ def register_maps_in_space_time_dataset(
     start_time_in_file = False
     end_time_in_file = False
     semantic_label_in_file = False
+    element = {
+        "rast": "cell",
+        "raster": "cell",
+        "rast3d": "grid3",
+        "raster3d": "grid3",
+        "raster_3d": "grid3",
+        "vector": "vector",
+    }[type]
```

Comment:
How comes it worked before without this?

Reply:
It worked because

  a) the `find_file` function replaces "raster" with "cell" (and then gives a message if in verbose mode that the user may not be able to understand, see: https://github.com/OSGeo/grass/blob/ca3b0a8f2b1109867c2034dc41fe3b146ef7b447/python/grass/script/core.py#L1345-L1347) and
  b) if a 3d map was not found (which I guess never was the case) the function simply assumed it is in the current mapset, which may be true for the tests.

Reply:
Makes sense, but perhaps best hidden in a library function which actually understands the user-facing types and does not complain about them, but that's for a different PR.

Reply:
Addressed in `find_file`